### DISCUSSION TITLE
添加角色攻略

### DISCUSSION THREAD

**Discussion by f5723508** [Arondight/Adachi-BOT#601] [General]
![Screenshot_2022-01-23-17-37-12-201_com tencent mo](https://user-images.githubusercontent.com/65771380/150672999-7ea2cccb-2fd6-47be-b871-07ae21dd3ac7.jpg)
类似这种图片

**Comment by mark9804** [comment]
这个 bot 生成的米游社图看着眼熟，知道它叫什么名字吗

看 owner 的意见，攻略我自己没有意向加，因为它不中立，可能有误导性，我个人来说不希望有人被带偏了之后我被牵连着作为清算对象

**Reply by Arondight** [reply]
不加，不交别人玩游戏，bot 可以问  @MiniGrayGay ，这个频道 bot 他在运营的，算是一个缝合怪

**Reply by Arondight** [reply]
https://github.com/pcrbot/erinilis-modules
